cron.yaml

```diff
@@ -42,3 +42,7 @@ cron:
   - description: Send Developer reports to partners
     url: /publish/SendDevelopersReportEvent
     schedule: every monday 01:00
+
+  - description: backfill records on database
+    url: /publish/backfillRecordsEvent
+    schedule: every sunday 01:00
```

Comment:
why every sunday, isn't it meant to run once?

Reply:
is it possible to schedule a cron for one day only @godolatunji

Reply:
@MuthomiMate pls reach out to George, Samuel or Sharon for help.